### ISSUE TITLE
snapshot-timemachine file timestamp notes

### ISSUE THREAD

**Issue by kryptt** [j-keck/zfs-snap-diff#13]
Hi, 

It seems I failed to notice two things with the original code with #12. 
Appologies. 

the timestamp given for the record is the snapshot timestamp; is it possible to use the file timestamp instead? 
Also; parse-time-string would be happier with a RFC 822 datetime:

"Parse the time in STRING into (SEC MIN HOUR DAY MON YEAR DOW DST TZ).
STRING should be something resembling an RFC 822 (or later) date-time, e.g.,
\"Fri, 25 Mar 2016 16:24:56 +0100\", but this function is
somewhat liberal in what format it accepts, and will attempt to
return a \"likely\" value even for somewhat malformed strings.
The values returned are identical to those of `decode-time', but
any unknown values other than DST are returned as nil, and an
unknown DST value is returned as -1."

**Comment by j-keck** [comment]
Hey, 

i'm busy this week - but i will take look at it on the weekend.

**Comment by kryptt** [comment]
What I ended up doing in the interim was real the file attributes directly: 
https://github.com/mrBliss/snapshot-timemachine/pull/5/files#diff-e629dfd1557af6d626328a19d6eb6161R365-R366

**Comment by j-keck** [comment]
@kryptt sorry for the delay.

I created a new repo for [zsd](https://github.com/j-keck/zsd). 
The next release of `zfs-snap-diff`will be without `zsd` - so i can create releases independently.

I have implemented your suggestions in j-keck/zsd@134aa38e and released the new version: **v1.1.2**.
You can get it from the [zsd site](https://j-keck.github.io/zsd/docs/install/).

I'm closing this issue here - please [re-open the issue in the zsd repo](/j-keck/zsd/issues/1) if something is missing.

Thanks

**Comment by j-keck** [comment]
example output:
```
./zsd -snapshot-timemachine main.go list
0	zfs-auto-snap_hourly-2020-07-01-13h00U	/home/.zfs/snapshot/zfs-auto-snap_hourly-2020-07-01-13h00U/j/prj/priv/zfs-snap-diff/zsd/main.go	Sun, 28 Jun 2020 19:55:54 +0200
```
